source/_patterns/04-components/message/message.md

```diff
@@ -2,9 +2,8 @@
 el: .message
 title: Message
 ---
-_Component for Message_
 
-## Variables
-* type: [string] Type of message.
+__Variables:__
+* type: [string] Type of the message (error, warning, or status).
```

Comment:
I think this should be "The type of message." Good call to list the allowed types here, since it's a limited set.